packages/backend/src/managers/applicationManager.ts

```diff
@@ -126,15 +126,22 @@ export class ApplicationManager {
       throw new Error('Cannot load configuration file.');
     }
 
-    // Mark as success.
-    loadingConfiguration.state = 'success';
-    taskUtil.setTask(loadingConfiguration);
-
     // Filter the containers based on architecture
     const filteredContainers = aiConfig.application.containers.filter(
       container => container.arch === undefined || container.arch === arch(),
     );
 
+    if (filteredContainers.length > 0) {
+      // Mark as success.
+      loadingConfiguration.state = 'success';
```

Comment:
I would create a new task. If there is no container for the user arch/gpu it is not that the config file failed at loading but that it does not support the user system. I would create one to specify it. Like checking if system supports app or something like that.

Reply:
This is a possibility that works, but on the long term I was thinking supporting an optional "error" property on the task, where we could provide more insight on the reason of a failing task.

Reply:
TBH I don't know which is better but if others do not complain againts it you can go and merge.

Reply:
I think we should probably either way, start using the Taks from the podman-desktop api and just proxy them, but the api is kinda limiting for now, but discussion open